packages/widget_toolkit/example/linux/flutter/ephemeral/.plugin_symlinks/url_launcher_linux

```diff
@@ -0,0 +1 @@
+/Users/pwndp/.pub-cache/hosted/pub.dev/url_launcher_linux-3.1.1/
```

Comment:
Do we really need to add user specific configurations such as /Users/pwndp ?

Reply:
Done!